packages/simplebar-react/index.js

```diff
@@ -35,7 +35,14 @@ const getOptions = function(obj) {
 };
 
 const SimpleBar = React.forwardRef(
-  ({ children, scrollableNodeProps = {}, ...otherProps }, ref) => {
+  ({ children, scrollableNodeProps = {}, tag = 'div', ...otherProps }, ref) => {
+    const RootTag = tag;
+    if (
+      typeof document === 'object' &&
+      document.createElement(RootTag) instanceof HTMLUnknownElement
+    ) {
+      throw new Error('Invalid tag name');
```

Comment:
Do you think this safeguard is really necessary? I would think that if user wants to pass an invalid/custom HTML element that's their problem. Also I don't know in what context but eventually they might want to use a [web component custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements)?
Since we are trying to keep the code as small/simple as possible.